DESCRIPTION

```diff
@@ -74,6 +74,7 @@ Imports:
 Suggests:
     servr,
     sf,
+    sp,
```

Comment:
@ampurr Need to list sp in Suggests in order for it to be installed and available for tests during github actions.

Reply:
@tdhock: Oh, thanks! Didn't realize.